### PR TITLE
[progress] support multiple ProgressPlugins

### PR DESCRIPTION
**Pull request by gamerson**
Signed-off-by: Gregory Amerson <gregory.amerson@liferay.com>

### CODE REVIEW

File: biz.aQute.bndlib/src/aQute/bnd/http/HttpClient.java [hunk 1]
```diff
@@ -265,13 +263,49 @@ public TaggedData call() throws Exception {
...
+			final List<ProgressPlugin.Task> multiplexedTasks = new ArrayList<>();
+
+			for (ProgressPlugin progressPlugin : progressPlugins) {
+				multiplexedTasks.add(progressPlugin.startTask(name, size));
+			}
+
 			task = new ProgressPlugin.Task() {
+				@Override
+				public void worked(int units) {
+					for (ProgressPlugin.Task multiplexedTask : multiplexedTasks) {
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/http/HttpClient.java]: To be nitpicky, the loop variable should be called `task` so it is the same as the other method's loops.

File: biz.aQute.repository/src/aQute/bnd/deployer/repository/ProgressWrappingStream.java [hunk 2]
```diff
@@ -12,8 +15,60 @@
...
+			final List<ProgressPlugin.Task> multiplexedTasks = new ArrayList<>();
+
+			for (ProgressPlugin progressPlugin : progressPlugins) {
+				multiplexedTasks.add(progressPlugin.startTask(name, size));
+			}
+
+			task = new ProgressPlugin.Task() {
+				@Override
+				public void worked(int units) {
+					for (ProgressPlugin.Task multiplexedTask : multiplexedTasks) {
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/deployer/repository/ProgressWrappingStream.java]: To be nitpicky, the loop variable should be called `task` so it is the same as the other method's loops.

File: biz.aQute.repository/src/aQute/bnd/deployer/repository/ProgressWrappingStream.java [hunk 3]
```diff
@@ -12,8 +15,60 @@
 	private Task		task;
 
 	public ProgressWrappingStream(InputStream delegate, String name, int size, ProgressPlugin progressPlugin) {
+		this(delegate, name, size, Collections.singletonList(progressPlugin));
```
Comment (bjhargrave) [inline biz.aQute.repository/src/aQute/bnd/deployer/repository/ProgressWrappingStream.java]: What if the arg `progressPlugin` is null? This method creates a single element list with null which will blow up in the other constructor which assumes the list holds real values.

**Comment by gamerson** [comment]
Thanks for review, pushed new commit.

**Comment by bjhargrave** [comment]
LGTM

**Pull request merged**